pkg/i18n/translations.go

```diff
@@ -80,11 +80,25 @@ func Errorf(ctx context.Context, format string, a ...interface{}) error {
 }
 
 func init() {
+	locale := os.Getenv("KUBEGEMS_LOCALE")
```

Comment:
It is worth discussing whether to control the Locale configuration directly through environment variables or through options